less/pages/events.less

```diff
@@ -13,6 +20,40 @@
 }
 
 .mailinglist-signup {
+  max-width: 50rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: left;
+  @media screen and (min-width: @screen-sm-min) {
+    flex-direction: row;
+    align-items: stretch;
+  }
+  .fieldset-container {
+    width: 100%;
+    @media screen and (min-width: @screen-sm-min) {
+      width: 35rem;
+      margin-right: 1rem;
+    }
+  }
+  .btn-container,
+  input[type="submit"] {
+    width: 100%;
+    @media screen and (min-width: @screen-sm-min) {
+      width: auto;
+    }
+  }
+  input {
+    padding: 1rem 1.5rem;
+    border: .1rem solid @grey;
+  }
+  input[type="email"] {
+    width: 100%;
+  }
+  input[type="submit"] {
+    margin-top: 0;
+  }
```

Comment:
we should find time to unifying our form stylings across the site though... :eyes: